packages/nimbus/src/components/text-input/text-input.tsx

```diff
@@ -32,12 +32,7 @@ export const TextInput = forwardRef<HTMLInputElement, TextInputProps>(
     const { inputProps } = useTextField(otherProps, ref);
 
     return (
-      <TextInputRootSlot
-        {...recipeProps}
-        {...styleProps}
-        {...otherProps}
```

Comment:
Do we even need `otherProps` as it's processed into `inputProps` by `useTextField`?

I opted to remove it. We can also include it via something like
```
const mergedProps = { ...otherProps, ...inputProps };
```
, but order matters in this case and it feels like a brittle gotcha moment.

Reply:
I dont believe we need `otherProps` here, but if we run into issues later on we can come back to that assumption